### PR TITLE
feat(composer): add hook to convert data bindings to queries

### PR DESCRIPTION
**Pull request by sheilaXu**
## Overview
add hook to convert data bindings to queries

### Scene Composer
For `scene-composer` package changes specifically, you can preview the component in the published storybook artifact. To do this, wait for the `Publish Storybook` action to complete below.

- Click on the workflow details
- Select the Summary item on the left
- Download the zip file

To run the storybook build locally, you need a local static web server:

```
npm install -g httpserver
cd <Extracted Zip Directory>
httpserver
```

Then open the website http://localhost:8080 to run the doc site.

## Legal
This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

File: packages/scene-composer/src/hooks/useBindingQueries.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import { useStore } from '../store';
+
+/**
+ * Create query objects for data bindings.
+ *
+ * @param bindings the data bindings to create query object
+ * @returns `queries` as an array of query objects for each data binding in the same order.
+ *          If the query object cannot be created for any of the bindings,
+ *          undefined will be returned in the result array at the same index.
+ */
```
Comment (tracy-french) [inline packages/scene-composer/src/hooks/useBindingQueries.ts:16]: Nice doc comment!

File: packages/scene-composer/src/hooks/useBindingQueries.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { useMemo } from 'react';
+import { Query, TimeSeriesData } from '@iot-app-kit/core';
+import { isEmpty } from 'lodash';
+
+import { useSceneComposerId } from '../common/sceneComposerIdContext';
+import { IValueDataBinding } from '../interfaces';
+import { useStore } from '../store';
+
+/**
+ * Create query objects for data bindings.
```
Comment (tracy-french) [inline packages/scene-composer/src/hooks/useBindingQueries.ts:10]: What does data binding refer to in this context? It's a general term yet seems to be referring to a specific data type utilized by the system. Would it be helpful to expand the comment here with additional info into what the data bindings are?

**Review by tracy-french — APPROVED** [review]
Thank you, @sheilaXu. :)

File: packages/scene-composer/src/hooks/useBindingQueries.ts (hunk 2 above)
Comment (sheilaXu) [reply packages/scene-composer/src/hooks/useBindingQueries.ts:10]: Data binding is a specific feature in the context of the scene composer, or even in TwinMaker. It is a set of ids (currently support entityId, componentName, propertyName) to uniquely identify a data to be used by scene widgets to change their visual state. This hook is supposed to be used internally only, so whoever use it should know the definition.

**Pull request merged**